### DISCUSSION TITLE
关于legend分页的疑问

### DISCUSSION THREAD

**Discussion by cuidong626** [antvis/G2#5873] [Q&A]
### 问题描述

如何使用legend的分页功能？
下面链接是官方的演示demo，缩小宽度增加如下代码.
`  .legend('color', {
    itemMarker: 'circle',
    maxRows:1
  })`
https://g2.antv.antgroup.com/examples/component/label#contrastReverse

![image](https://github.com/antvis/G2/assets/22650893/ef1b44d7-6a5e-4f14-bd38-a4b69dbcdbc4)
观察到 legend进行了分页，每页占一行。

接下来增加一个paddingTop的配置，
`paddingTop:100`
此时发现每页占了两行？这是什么问题？
![image](https://github.com/antvis/G2/assets/22650893/c82d0e5a-eb58-4da3-a15d-5b79bb119391)

### 重现链接

https://codesandbox.io/p/sandbox/blissful-merkle-6t8frz?file=%2Findex.ts

### 预期行为

1.期望paddingTop 不会影响的分页所占的行数

### 平台

- 操作系统: [macOS, Windows, Linux, React Native ...]
- 网页浏览器: [Google Chrome, Safari, Firefox]

**Comment by pearmini** [comment]
padding 指定的是组件绘制区域大小，组件会自适应，你可以用 marginTop
